src/safe-upgrade-tool/prerequisites.md

```diff
@@ -0,0 +1,22 @@
+---
+group: software-update-guide
+title: Prerequisites
+functional_areas:
+  - Upgrade
+---
+
+Running the Safe Upgrade Tool (SUT) helps you identify what you must do **before** upgrading Magento.
+
+The minimum requirements to run the SUT are:
+
+| **Requirements** | **Constraints** |
+|----------------|-----------------|
+| PHP version| >= 7.3 |
+| Composer | none |
+| Node.js | none |
```

Comment:
Not JS is optional, and is only needed for verifying GraphQL backward compatibility testing.